cql3/restrictions/multi_column_restriction.hh

```diff
@@ -376,17 +376,24 @@ protected:
 };
 
 class multi_column_restriction::slice final : public multi_column_restriction {
-    using restriction_shared_ptr = ::shared_ptr<clustering_key_restrictions>;
+public:
+    enum class mode : char {
+        normal,
+        scylla_clustering_bound,
```

Comment:
Can we please have a blurb on this, even if it references the API documentation?

Reply:
Blurb as in md docs, or blurb as in explanatory comment?

Reply:
I meant explanatory comment.